x/cdp/abci.go

```diff
@@ -36,6 +39,14 @@ func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k keeper.Keeper)
 			panic(err)
 		}
 
+		if blockHeight%params.BeginBlockerExecutionBlockInterval != 0 {
```

Comment:
This is probably a good case for a named parameter now since it's duplicated in a loop and we are already storing the block height in a variabled

```go
skipSyncronizeAndLiquidations := ctx.BlockHeight()%params.BeginBlockerExecutionBlockInterval != 0

...

  if skipskipSyncronizeAndLiquidations {
  ...
  }
```

Reply:
good idea, updated